app/src/main/java/ch/epfl/sweng/zuluzulu/Fragments/MementoFragment.java

```diff
@@ -101,9 +116,55 @@ private void handleMemento(List<String> result) {
             addEvent(datas);
         }
 
+        addDatabase();
         IdlingResourceFactory.decrementCountingIdlingResource();
     }
 
+    private void addDatabase(){
```

Comment:
This function should be reformated xD

Reply:
haha it's fine

Waiting for a good proxy

Oh and thanks Gaultier... Test are not passing because he hard coded events......... -_-